src/components/PrivateRoute/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 import { Route, Redirect } from 'react-router-dom'
+import { connect } from 'react-redux'
 
 // TODO: connect to global state
-const isAuthenticated = false
+const isAuthenticated = true
```

Comment:
Since you injected `user` object as props to this component, you should use it instead of hard-coded boolean flag to determine if user is authenticated or not